spec/features/trainee_actions/defer_trainee_spec.rb

```diff
@@ -63,7 +64,45 @@
     end
   end
 
+  scenario "course start date is in the future" do
+    given_i_initiate_a_deferral(commencement_date: nil, course_start_date: Time.zone.today + 1.day)
```

Comment:
Happy to go along with this, but we are trying to avoid lines having arguments. A sentence describing the state is preferable.

Reply:
Updated in https://github.com/DFE-Digital/register-trainee-teachers/pull/1737/commits/679af94d20b0a8ddb3f6bbe368f060174ba29910, is that better?